caom2-search-server/src/main/java/ca/nrc/cadc/caom2/ADQLGenerator.java

```diff
@@ -378,6 +379,7 @@ String getSelectList(final String utypeSelectList) {
             // Upload target, ra and dec come in to the service as part of the
             // list of requested columns. Only radius needs to be added.
             sb.append(prefix + SEARCH_UPLOAD_TABLE + ".radius");
+            sb.append(prefix + SEARCH_UPLOAD_TABLE + ".position");
```

Comment:
Would you please fix lines 381 and 382 to be proper StringBuilder.append() calls?  Building the string inside the append defeats the purpose of it.  Thanks.

Also fix the comment above it.

Reply:
ok